### ISSUE TITLE
About the transformer.

### ISSUE THREAD

**Issue by CGLemon** [QueensGambit/CrazyAra#188]
Hi Johannes Czech!

I notice that your team uses the transformer in the training pipe. I am curious about current result.

I tried to do the same thing on the 9x9 go game too. My result showed the transformer was not good at go game if we did not any special normalizing or encoding for inputs features. Or just I did not give it enough training data. I have no ideal....?

It is very helpful to me if you could share some current result. Thanks a lot!

**Comment by QueensGambit** [comment]
Hello @CGLemon ,

yes, I've been doing some experiments with transformers over the last months.
I can confirm that various published vision transformers didn't perform well with respect to measured latency on TensorRT.
However, I had some success at building a hybrid transformer.
We are currently in the process of writing a paper about it that we hope to publish soon.

I will get back to you as soon as the preprint is available on ArXiv.

Kind regards,
Johannes Czech

**Comment by QueensGambit** [comment]
Hello @CGLemon ,
I have just sent you an email to your registered GitHub email address with some preliminary information about the upcoming paper.

**Comment by CGLemon** [comment]
Thank you for your information. I am awaiting the paper!

**Comment by QueensGambit** [comment]
A preprint of our paper has been published on arXiv:
https://arxiv.org/abs/2304.14918
